common/lib/client/resource.js

```diff
@@ -1,5 +1,8 @@
 var Resource = (function() {
 	var msgpack = (typeof(window) == 'object') ? window.Ably.msgpack : require('msgpack-js');
+	var isErrATokenProblem = function(error) {
```

Comment:
This wasn't renamed, and is also a duplicate. Can we not put it into `ConnectionManager.isTokenErr()` ?